recipes/runtimeqml/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake
+from conan.tools.files import get, copy
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.49.0"
+
+class RuntimeQml(ConanFile):
+    name = "runtimeqml"
+    homepage = "https://github.com/GIPdA/runtimeqml"
+    description = "Enables hot-reloading qml files"
+    topics = ("qt", "hot-reload", "qml", "gui")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "CMakeDeps", "CMakeToolchain"
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, self.export_sources_folder)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][str(self.version)],
+            destination=self.source_folder, strip_root=True)
+
+    def requirements(self):
+        if self.version == "qt5":
+            self.requires("qt/[>=5.0.0 <6.0.0]")
```

Comment:
Ranged version is forbidden: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-are-version-ranges-not-allowed